scene/3d/bone_attachment_3d.cpp

```diff
@@ -52,55 +60,321 @@ void BoneAttachment3D::_validate_property(PropertyInfo &property) const {
 	}
 }
 
+bool BoneAttachment3D::_set(const StringName &p_path, const Variant &p_value) {
+	if (p_path == "override_pose") {
```

Comment:
You can now use the `SNAME` macro to define statically compiled `StringName`, this will avoid creating the `StringName`s at runtime, each time `_set` is called:
```c++
if (p_path == SNAME("override_pose")) {
```